meson.build

```diff
@@ -229,7 +242,13 @@ endif
 
 # handle resources
 nntrainer_resdir = meson.build_root() / 'res'
-run_command('mkdir', '-p', nntrainer_resdir)
+
+if host_machine.system() == 'windows'
+  nntrainer_resdir_win = nntrainer_resdir.replace('/', '\\')
```

Comment:
Can't we construct it correctly using '/'?
Couldn't test with wine since `Z:\` is required for abs path.